engine/src/main/resources/assets/prefabs/player/player.prefab

```diff
@@ -20,13 +20,6 @@
   },
   "VisualCharacter": {
   },
-  "BoxShape": {
-    "extents": [
-      1.5,
-      1.8,
-      1.5
-    ]
-  },
```

Comment:
do we still need the BoxShape and does it still serve a purpose? I remember there was some discussion about the there being two colliders on the player.

Reply:
yeah, we discussed this on Discord - nobody seems to know what this is used for, and the numbers are off from the capsule shape used otherwise for the player. 
People suggested to remove it now - we can easily bring it back if we encounter some bugs (hard part will be to connect these potential bugs with the lack of a `BoxShapeComponent` on the player, though)